### DISCUSSION TITLE
Pkg.activate gets executed after using

### DISCUSSION THREAD

**Discussion by VarLad** [fonsp/Pluto.jl#1006] [Pluto development]
I think I see a trivial (but annoying) problem when using Pluto
`using` cells are executed before `import` cells
as a result,
```
import Pkg;
Pkg.activate("/a/b/c")
```
cell gets executed _after_ `using xyz` cell, when I first run my notebook
causing the xyz not found error
I think there should be a command in Pluto, something like `activate"/a/b/c"`, which gets executed before any other cell, when a notebook is started

**Comment by fonsp** [comment]
This is fixed on Pluto#main, and will be released soon.

**Reply by fonsp** [reply]
Until then, use a `begin` block with all your Pkg calls and package imports
